pkg/jobcreator/run.go

```diff
@@ -46,20 +49,42 @@ func RunJob(
 	// wait a short period because we've just started the job creator service
 	time.Sleep(100 * time.Millisecond)
 
+	// Start run job trace
+	c, span := jobCreatorService.controller.tracer.Start(ctx.Ctx, "run_job",
+		trace.WithAttributes(
+			attribute.String("job_offer.job_creator", offer.JobCreator),
+			attribute.String("job_offer.module.repo", offer.Module.Repo),
+			attribute.String("job_offer.module.hash", offer.Module.Hash),
+			attribute.String("job_offer.mode", string(offer.Mode)),
```

Comment:
We will likely want to add more fields around pricing when we continue our work there.